src/amchart/CommonSerial.js

```diff
@@ -506,13 +506,10 @@
             gObj.valueAxis = "v0";
         }
 
-        gObj.balloonFunction = function(d) {
-            //var balloonText = d.category + ", " + context.columns()[d.graph.columnIndex+1]  + ": " + context.data()[d.index][d.graph.columnIndex+1];
-            if (d.graph.type === "line") {
-                return d.category + ", " + context.columns()[d.graph.index + 1]  + ": " + context.data()[d.index][d.graph.index + 1];
-            } else {
-                return d.category + ", " + context.columns()[d.graph.columnIndex + 1]  + ": " + context.data()[d.index][d.graph.columnIndex + 1];   
-            }
+        gObj.balloonFunction = function(item, graph) {
+            var key = graph.valueField;
+            var data = graph.chart.dataProvider;
+            return item.category + ", " + key + ": " + data[item.index][key];
```

Comment:
I don't believe so (only if there was no bar chart displayed) - I tested on as many samples as I could and it looked good.  
There is an identical issue with click I just spotted, so will open an issue for that as well.